<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>841</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>809</width>
      <height>839</height>
     </rect>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>791</width>
       <height>180</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>791</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>791</width>
       <height>201</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>789</width>
        <height>178</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget_unidades_construccion">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>161</height>
        </rect>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <column>
        <property name="text">
         <string>Id unidad construcción</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Identificador</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="tableWidget_interesados">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>361</width>
         <height>161</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Calificaciones</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Info_complementaria">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>791</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>791</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>791</width>
       <height>541</height>
      </size>
     </property>
     <property name="title">
      <string>Detalle Construcción</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Número semisótanos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_semisotanos">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>69</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Identificador</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_identificador">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Número de pisos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_pisos">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Número de sótanos</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tipo_construccion">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>210</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dominio_tipocons">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_rel_superficie">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QgsMapCanvas" name="wdt_mapa_cons" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>471</width>
        <height>400</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>381</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_sotanos">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Tipo de construcción</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Dominio del tipo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>115</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Relación superficie</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>t_id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_t_id">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Número mezanines</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>40</y>
        <width>35</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Altura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>90</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Año construcción</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Avalúo Comercial</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Avalúo Catastral</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_mezanines">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_area_construccion">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_anio_construccion">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Avaluo_Catastral_cons">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Avaluo_Comercial_cons">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Área construcción</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_altura">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_cerrar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>810</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cerrar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapCanvas</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
